kategory/src/main/kotlin/kategory/arrow/Function1.kt

```diff
@@ -4,7 +4,7 @@ fun <I, O> ((I) -> O).k(): Function1<I, O> = Function1(this)
 
 operator fun <I, O> Function1Kind<I, O>.invoke(i: I): O = this.ev().f(i)
 
-@higherkind class Function1<I, out O>(val f: (I) -> O) : Function1Kind<I, O> {
+@higherkind open class Function1<I, out O>(val f: (I) -> O) : Function1Kind<I, O> {
```

Comment:
Nobody. It was being inherited by `Lifted` in the beginning, but no need for it anymore.